### DISCUSSION TITLE
v2.0.0-beta.2

### DISCUSSION THREAD

**Discussion by ALTaleX531** [ALTaleX531/TranslucentFlyouts#49] [Announcements]
1. Fix gradient color converting issue.
2. Fix possible crash of pop-in animation.
3. Add translucent support for tooltip.
4. Add support for stopping service temporarily.

If you want to disable tooltip handling please set dword `HKEY_CURRENT_USER\SOFTWARE\TranslucentFlyouts\Tooltip\Disabled` to 1.
No exceptional changes to config, you can simply treat Tooltip as DropDown.
**Full Changelog**: https://github.com/ALTaleX531/TranslucentFlyouts/compare/v2.0.0-alpha.4.2...v2.0.0-beta.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ALTaleX531/TranslucentFlyouts/releases/tag/v2.0.0-beta.2'>v2.0.0-beta.2</a>.</em>

**Comment by ghost** [comment]
for some reason this update doesnot work at all for me....i replaced the old one and after the restart it gives Rundll32.exe error. This has never happened before.
![Screenshot 2023-08-18 195017](https://github.com/ALTaleX531/TranslucentFlyouts/assets/100816109/2330ace9-a8cb-4944-9816-19d1f8971a54)

**Reply by ALTaleX531** [reply]
If you want to upgrade alpha.4.x to beta.2, you should also run `install.bat`. Because an extra parameter `/start` is required in this version.

**Reply by ghost** [reply]
thanks....its strange how i am the only one facing this issue while previous versions worked correctly....I have tried few times already since yesterday....Also tried completely uninstalling the previous one and installing the new one....and it works after it is installed but if I restart the pc then it displays that same message again. Stopping the service also works.....I also checked a new task scheduler was created after uninstalling and installing the new update.

**Reply by ghost** [reply]
thanks for the reply.....i now see the typo you might have made in the taskscheduler. it was /startup in my machine....edited to /start and its working well....thanks....also i like the way we can just stop the process now.

**Reply by ALTaleX531** [reply]
I see now, it was a bug that I didn't change the installer code entirely.

**Reply by ghost** [reply]
I want to say thanks again for making the dull start menu look lively....I also pair your mod with ExplorerBlurMica, OpenShell, Explorer Patcher, UltraUXThemePatcher and SecureUXTheme with custom theme and it still works perfectly.....it might be some time until microsoft finally make the context menu usable but until then this will be a great experience with TF. Also I might not update past windows 11 21H2 on this laptop so this laptop may see the end of the light accompanied by your work. Please slowly push out the updates....I have yet to face any real bug so this program of yours is working very well.
Edit: TranslucentFlyouts completes my machine
![Screenshot 2023-08-18 222854](https://github.com/ALTaleX531/TranslucentFlyouts/assets/100816109/e25615e5-37bb-4180-b936-69d82a7d05c8)

**Comment by A-Basir** [comment]
Hey guys
I wanted to ask about x86 version
I'm little confused by saying it's optional
Does that complete the x64 version or is it intended to be used on x86 systems?

**Reply by ALTaleX531** [reply]
64-bit build can only be effective for 64-bit app, 32-bit build can only be effective for 32-bit app.
Nowadays most of the apps are 64-bit version but there are still some 32-bit apps like IDM, VMware which require 32-bit build, so it is optional.
